demo/demo_secondary.py

```diff
@@ -53,7 +53,6 @@
 _primary_host = demo.PRIMARY_SERVER_HOST
 _primary_port = demo.PRIMARY_SERVER_DEFAULT_PORT
 firmware_filename = 'secondary_firmware.txt'
-current_firmware_fileinfo = {}
```

Comment:
I noticed that this variable still exists in [demo_primary](https://github.com/awwad/uptane/blob/21639e667be86d9d4741a9aa5560bf3aed4a4da7/demo/demo_primary.py#L80) but is never used.